edn_format/edn_dump.py

```diff
@@ -60,17 +60,51 @@ def seq(obj, **kwargs):
     return ' '.join([udump(i, **kwargs) for i in obj])
 
 
+def _indent_objs(objs, open_symbol, close_symbol, indent, indent_step, **kwargs):
```

Comment:
The naming of this function should IMHO reflect the fact it’s only used to dump collections. Something like `_udump_indent_collection` + a docstring.